x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/search_index.tsx

```diff
@@ -305,13 +304,7 @@ interface ContentProps {
   tabs: EuiTabbedContentTab[];
 }
 
-const Content: React.FC<ContentProps> = ({
-  config,
-  errorConnectingMessage,
-  index,
-  tabs,
-  tabId,
-}) => {
+const Content: React.FC<ContentProps> = ({ index, tabs, tabId }) => {
```

Comment:
You might want to remove `errorConnectingMessage` from  `ContentProps`

Reply:
OK done.